### ISSUE TITLE
操作耗时统计

### ISSUE THREAD

**Issue by ljdang** [fengjian0106/hed-tutorial-for-document-scanning#21]
利用TensorFlow Lite进行部署，BN消耗的计算时间很少，反倒是transposed convolution占用了大部分的时间，32位和64位的耗时也差不太多，统计数据如下：
![screenshot from 2018-11-05 19 43 59](https://user-images.githubusercontent.com/21071150/47996504-bf11da00-e133-11e8-875d-4108713d9daa.png)

**Comment by fengjian0106** [comment]
赞，终于可以用 TensorFlow Lite 部署了

**Comment by ljdang** [comment]
@fengjian0106 看您给出的统计数据，transposed convolution操作应该是耗时很少的，难道是TF Lite没做优化？这还是在骁龙845上的数据，离实时性要求还有点距离。

**Comment by fengjian0106** [comment]
嗯，有可能是 transposed convolution 优化的还不够好，我印象中以前在 android 上用 32bit 环境测试的时候，batch norm 的耗时就比 64bit 上多了很多。

**Comment by dustturtle** [comment]
@ljdang 在tflite上部署要做哪些调整？能否分享一下，感谢

**Comment by Codermanok** [comment]
@ljdang   你好，请问你这是在哪个版本的tensorflow的tflite上profile的各个op时间呀？感觉和我测的以及作者的不太一样，我使用的是tensorflow-1.11.0，也是Android端，测的也是Conv2D耗时比例最高。下图是作者的，谢谢

<img width="954" alt="屏幕快照 2019-04-12 下午8 20 16" src="https://user-images.githubusercontent.com/7951309/56036969-c089a300-5d60-11e9-860e-b4e51cc0f676.png">
